src/components/carousel/carousel.js

```diff
@@ -38,6 +38,17 @@ class Carousel extends React.Component {
     ]),
 
     /**
+     * The selected slide
+     *
+     * @property slideIndex
+     * @type {String}
+     */
+    slideIndex: PropTypes.oneOfType([
```

Comment:
No, this will only be used when we want to manually set the slide index.
If the props is not set, the initialSlideIndex will be used when rendering the component, with a default value of 0.